payments_service/braintree/templates/braintree/emails/subscription_charged_unsuccessfully.html

```diff
@@ -0,0 +1,38 @@
+{# subscription header is added automatically here #}
+
+    <div class="padded">
+        <div class="error">Subscription charge failed</div>
```

Comment:
Also I'd suggest making this `class="banner error"`. `.banner` should have all the rounded corners and then error can just be a color modifier.